git-changebar/data/Makefile.rel.am

```diff
@@ -0,0 +1,4 @@
+if ENABLE_GITCHANGEBAR
+gcbdatadir = $(plugindatadir)/git-changebar
+dist_gcbdata_DATA = %D%/prefs.ui
```

Comment:
I'd really rather use the real canonical name instead of an abbreviated version, for consistency, clarity, etc.